ccloud/beginner-cloud/cleanup.sh

```diff
@@ -27,13 +27,21 @@ CLUSTER_NAME="demo-kafka-cluster"
 CLUSTER=$(ccloud kafka cluster list | grep $CLUSTER_NAME | tr -d '\*' | awk '{print $1;}')
 #echo "CLUSTER: $CLUSTER"
 CLIENT_CONFIG="/tmp/client.config"
+CONNECTOR_NAME="datagen_ccloud_pageviews"
+CONNECTOR=$(ccloud connector list | grep $CONNECTOR_NAME | tr -d '\*' | awk '{print $1;}')
 
 ##################################################
 # Cleanup
-# - Delete the Kafka topics, Kafka cluster, environment, and the log files
+# - Delete the managed Kafka connector, Kafka topics, Kafka cluster, environment, and the log files
 ##################################################
 
-echo -e "\n# Cleanup: delete topics, kafka cluster, environment"
+echo -e "\n# Cleanup: delete connector, topics, kafka cluster, environment"
+if [[ ! -z "$CONNECTOR" ]]; then
+    echo "ccloud connector delete $CONNECTOR"
+    ccloud connector delete $CONNECTOR 1>/dev/null
+    sleep 5
```

Comment:
Is the sleep necessary?

Reply:
I just ran through the demo twice without it and everything appears to be operating as expected. I'll remove the sleep.